lib/screens/edit_product_screen.dart

```diff
@@ -86,6 +98,49 @@ class _EditProductScreenState extends State<EditProductScreen> {
     );
   }
 
+  String validatorImageUrl(String value) {
+    if (value.isEmpty) {
+      return 'Please enter an image URL';
+    }
+    if (value.startsWith('http') && !value.startsWith('https')) {
+      return 'Please enter a valid URL.';
+    }
+    if (!value.endsWith('.png') && !value.endsWith('jpg') && !value.endsWith('.jpeg')) {
+      return 'Please enter a valid image URL.';
+    }
+    return null;
+  }
+
+  String validatorDescription(String value) {
+    if (value.isEmpty) {
+      return 'Please enter a description';
+    }
+    if (value.length < 10 != null) {
+      return 'Should be at least 10 characters long.';
+    }
+    return null;
+  }
+
+  String validatorPrice(String value) {
+    if (value.isEmpty) {
+      return 'Please enter a price.';
+    }
+    if (double.tryParse(value) == null) {
+      return 'Please enter a valid number';
+    }
+    if (double.parse(value) <= 0) {
+      return 'Please enter a number greater than zero.';
+    }
+    return null;
+  }
+
+  String validatorTitle(String value) {
+    if (value.isEmpty) {
+      return 'Please provide a value.';
+    }
+    return null;
```

Comment:
can you try if this is possible:

`return value.isEmpty ? 'Please provide a value' : null;`

Reply:
it worked.